src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -401,6 +401,20 @@ protected function createChildContext(array $parentContext, $attribute/*, string
     private function getCacheKey($format, array $context)
     {
         unset($context['cache_key']); // avoid artificially different keys
+
+        if (interface_exists(\Throwable::class)) {
+            try {
+                return md5($format.serialize([
+                        'context' => $context,
+                        'ignored' => $this->ignoredAttributes,
+                        'camelized' => $this->camelizedAttributes,
+                    ]));
+            } catch (\Throwable $exception) {
```

Comment:
`\Exception`?
catching `\Throwable` is a terrible practice, it hides coding issues and help ppl lose their time to figure out what is broken...

Reply:
Not here, it's a very specific edge case, see curl https://fabbot.io/patch/symfony/symfony/36336/ea24bfca20b84a117f032ed9367b3aa1ef43d548/cs.diff | patch -p0

Reply:
If we don't do that, if any value in the context contains a Doctrine entity, it an error can be thrown, leading to a 500.